fuse_core/test/test_uuid.cpp

```diff
@@ -146,6 +149,59 @@ TEST(UUID, Generate)
   }
 }
 
+void generateUUIDs(UUIDs& uuids)
+{
+  constexpr size_t uuid_count = 100000;
+  uuids.reserve(uuid_count);
+  for (size_t i = 0; i < uuid_count; ++i)
+  {
+    auto uuid = fuse_core::uuid::generate();
+    uuids.push_back(uuid);
+  }
+}
```

Comment:
I think the general UUID generator does more work to ensure it is globally unique, which explains why it is slow. For test of 100K times, it may still get low probability to repeat, what if even larger numbers? So, this depends on application, what if there are duplicated UUID happens in the program? Any critical consequence?

Reply:
I was running tests locally with 1 billion UUIDs, but I didn't want the unit test run by the server to take 10+ minutes.

The original code was taken from the Boost random UUID implementation. I feel like they are literally just populating the UUID with random bits.

But yes, duplicates are bad. The UUID is used to determine if two things are the same. Incorrectly saying thing1 is the same as thing2 is a critical error.

Reply:
For whatever this random website I dug up is worth, it also says generate 16 bytes of random data.
https://www.cryptosys.net/pki/uuid-rfc4122.html

Reply:
Since it is extensively tested both locally and by community, it should be fine.